nautobot/extras/tests/example_jobs/test_soft_time_limit_exceeded.py

```diff
@@ -0,0 +1,14 @@
+import time
+from nautobot.extras.jobs import Job
+
+
+class TestSoftTimeLimitExceeded(Job):
+    class Meta:
+        name = "Soft Time Limit Exceeded"
+        description = "Set a soft time limit of 1 second`"
+        soft_time_limit = 1
+
+    def run(self, data, commit):
+        time.sleep(1.5)
```

Comment:
I think there's a bit of wiggle room in how precisely Celery enforces the time limits - to avoid spurious failures, maybe increase this to, say, `time.sleep(3)`?

Reply:
The test that tests this job does't actually work as expected. I should be able to get the string `SoftTimeLimitExceeded: SoftTimeLimitExceeded()` from the output of the management command (see output from manual run in attached image), but this isn't the case. I'll go ahead (later today) and delete this test job and the related automated test case as they're pointless.

<img width="1388" alt="Screenshot 2022-01-28 at 07 34 55" src="https://user-images.githubusercontent.com/655255/151506079-99e5c321-5ec2-4aec-a378-53149f32bc99.png">

Reply:
Tests don't actually test this properly so I'm removing them